src/shared/molecules/MyDataTable/MyDataTable.tsx

```diff
@@ -189,14 +190,15 @@ const MyDataTable: React.FC<TProps> = ({
         ellipsis: true,
         render: (text, record) => {
           const showedText = isValidUrl(text) ? text.split('/').pop() : text;
+          const resourceId = record['@id'] ?? record._self;
           if (text && record._project) {
             const { org, project } = makeOrgProjectTuple(record._project);
             return (
               <Tooltip title={text}>
                 <Button
                   style={{ padding: 0 }}
                   type="link"
-                  onClick={() => goToResource(org, project, text)}
+                  onClick={() => goToResource(org, project, resourceId)}
                 >
                   {showedText}
```

Comment:
i think `showedText` will be replaced with `text` since we are getting the right `name` now
and the Tooltip will get the `@id` if there is no name

Reply:
Thanks for catching that. I've made the change. Here's how it looks like now:

![Screenshot from 2023-06-12 10-05-34](https://github.com/BlueBrain/nexus-web/assets/11242410/2d6a7995-ecfe-41c9-89d0-7bc4c42b3215)
When a name was found


![Screenshot from 2023-06-12 10-05-55](https://github.com/BlueBrain/nexus-web/assets/11242410/1a956613-2113-45e1-9973-f6669c79755b)
When no name was found